Library/ViewModels/PledgeViewModel.swift

```diff
@@ -141,13 +141,15 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
     }
     .skipNil()
 
-    self.confirmationLabelAttributedText = Signal.merge(
+    let initialProjectAndProject = Signal.merge(
```

Comment:
Here the `initialProjectAndProject` are the same project, but emitted in different times and this can end up becoming misleading. Maybe something like this would be better?
```
 let projectAndPledgeTotal = Signal.merge(
      project,
      project.takeWhen(self.traitCollectionDidChangeSignal)
    )
    .combineLatest(with: pledgeTotal)

self.confirmationLabelAttributedText = projectAndPledgeTotal
    .ksr_debounce(.milliseconds(10), on: AppEnvironment.current.scheduler)
    .map(attributedConfirmationString(with:pledgeTotal:))
    .skipNil()
```

Reply:
Oh, I forgot to change that 🤦🏾‍♀️. Thanks for catching!